proposals/simd/docs/SIMD-sign-and-zero-extended-loads.md

```diff
@@ -0,0 +1,64 @@
+### **Proposal WebAssembly SIMD Modification**
```

Comment:
Can this document be removed? While this is useful information, I don't see a need for this to be included here.

Reply:
Do you want it to be included in the body of the PR or just simply remove it? @rrwinterton, what do you think?